modules/configuration/test/ConfigurationTest.php

```diff
@@ -41,5 +41,36 @@ public function testConfigurationPageLoads()
             ->findElement(WebDriverBy::cssSelector("body"))->getText();
         $this->assertContains("Configuration", $bodyText);
     }
+    /**
+     * Tests that configration loads with the permission
+     *
+     * @return void
+     */
+    function testConfigPermission()
+    {
+         $this->setupPermissions(array("config"));
+         $this->safeGet($this->url . "/configuration/");
+         $bodyText = $this->webDriver->findElement(
+             WebDriverBy::cssSelector("body")
+         )->getText();
+         $this->assertContains("Configuration", $bodyText);
```

Comment:
Are we sure that the word "Configuration" doesn't appear on the error page too (for the configuration module)?